frontend/js/tabs.js

```diff
@@ -0,0 +1,18 @@
+// set initial tab
+changeTab('encrypt');
```

Comment:
This is technically unnecessary because `index.html` already has the relevant styles applied, but it feels extra brittle to not have some initialization here.

Reply:
Good idea to do it.

Reply:
Fine with me